config/api.yml

```diff
@@ -621,6 +621,8 @@
         :identifier: embedded_configuration_script_source_delete
       - :name: create
         :identifier: embedded_configuration_script_source_add
+      - :name: refresh
+        :identifier: embedded_configuration_script_source_edit
```

Comment:
@dclarizio which product features should be used for refresh? There are a bunch of them out there, wasn't sure which one made sense here.